services/inc/LinkLossService.h

```diff
@@ -0,0 +1,126 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2020 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef BLE_LINK_LOSS_SERVICE_H
+#define BLE_LINK_LOSS_SERVICE_H
+
+#ifdef BLE_FEATURE_GATT_SERVER
+
+#include "ble/BLE.h"
+#include "ble/Gap.h"
+#include "events/EventQueue.h"
+#include "ble/gap/ChainableGapEventHandler.h"
+
+#include <chrono>
+
+/**
+ * BLE Link Loss Service
+ *
+ * @par purpose
+ *
+ * @par usage
```

Comment:
Purpose will give a basic overview of the service in line with the [LLS_v1.0.1](https://www.bluetooth.org/DocMan/handlers/DownloadDoc.ashx?doc_id=303198)
It might also be useful to add some insight into the local `EventHandler`. The user must override `onAlertRequested()` and `onAlertEnd()` to effectively interact with the user. 
What do you think?

Reply:
yes, my advice is to keep it short and link to the spec for more details